packages/top-nav/src/__stories__/TopNav.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import HigNotifications from "hig-react/lib/elements/components/GlobalNav/TopNav/Notifications";
+import Notification from "hig-react/lib/elements/components/GlobalNav/TopNav/Notification";
```

Comment:
As discussed, we'll need to revert https://github.com/Autodesk/hig/commit/d3bd6f70a67a44df588f7f24db013b0d0ea47ee5#diff-3d0cd8e00eb82d8272952de955a3b170 to restore this export.

Reply:
Dove in myself because I felt responsible. Let's try cherry-picking a3eec10.